### PR TITLE
Update versioncheck addon

### PR DESCRIPTION
**Pull request by razzeee**

### CODE REVIEW

File: addons/service.xbmc.versioncheck/lib/shellhandlerapt.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def _check_versions(self, package, update=True):
             if installed == "(none)":
                 installed = False
             if candidate == "(none)":
-                candidate = False
+                candiate = False
```
Comment (ronie) [inline addons/service.xbmc.versioncheck/lib/shellhandlerapt.py:59]: typo

File: addons/service.xbmc.versioncheck/lib/versions.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 from lib.common import log
 
 def compare_version(version_installed, versionlist):
-    # Create separate version lists
+    # Create seperate version lists
```
Comment (ronie) [inline addons/service.xbmc.versioncheck/lib/versions.py:22]: typo

File: addons/service.xbmc.versioncheck/lib/versions.py [hunk 3]
```diff
@@ -122,6 +122,6 @@ def compare_version(version_installed, versionlist):
                 version_available = versionlist_alpha[0]
                 oldversion = True
                 log('Version available  %s' %versionlist_alpha[0])
-                log('You are running an older non alpha version')     
+                log('You are running an older non alpah version')
```
Comment (ronie) [inline addons/service.xbmc.versioncheck/lib/versions.py:125]: typo

File: addons/service.xbmc.versioncheck/service.py [hunk 4]
```diff
@@ -52,7 +57,7 @@ def _versioncheck():
     versionlist = get_versionfilelist()
     # retrieve version installed
     version_installed = get_installedversion()
-    # compare installed and available
+    # copmpare installed and available
```
Comment (ronie) [inline addons/service.xbmc.versioncheck/service.py:60]: typo

**Comment by razzeee** [comment]
Thanks ronie!
There is a PR to fix those here: https://github.com/XBMC-Addons/service.xbmc.versioncheck/pull/8

**Pull request merged**